Scripts/release.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -e
+
+PROJECT=PINOperation
+
+# BRANCH=$(git rev-parse --abbrev-ref HEAD)
+# if [[ "$BRANCH" != "master" ]]; then
+#     echo "Please run on master branch."
+#     exit 1
+# fi
+
+if ! git diff-index --quiet HEAD --; then
+    echo "Please commit or stash any changes before running."
+    exit 1
+fi
+
+if [ -z "$GITHUB_CHANGELOG_API_KEY" ]; then
+    echo "Must set \$GITHUB_CHANGELOG_API_KEY environment variable"
+    exit 1
+fi
+
+case $1 in
+    "--major")
+        UPDATE_TYPE="major"
+        ;;
+
+    "--minor")
+        UPDATE_TYPE="minor"
+        ;;
+
+    "--patch")
+        UPDATE_TYPE="patch"
+        ;;
+    
+    *)
+        echo "Usage release.sh --patch / --minor / --major"
+        exit 1
+        ;;
+esac
+
+PODSPEC="$PROJECT.podspec"
+PODSPEC=`echo $PODSPEC`
```

Comment:
I'm not sure I understand what this line does.

Reply:
It used to do something when the above was `PODSPEC="*.podspec"` :)